Utils/ReleaseMaker.psm1

```diff
@@ -93,7 +93,6 @@ function New-ReleaseBuild
     try
     {
         remove-item out/ -recurse -force
-        dotnet restore
         .\buildCoreClr.ps1 -Framework net451 -Configuration Release -Build
         .\buildCoreClr.ps1 -Framework net451 -Configuration PSV3Release -Build
         .\buildCoreClr.ps1 -Framework netstandard1.6 -Configuration Release -Build
```

Comment:
should this be `netstandard2.<something>`?

Reply:
Why? The idea behind .net standard is to keep the number as low as possible. Unless we need an API that is available only in netstandard2.0, there is no value (the standard itself has nothing to do with the runtime, the runtime is responsible for implementing the standard). But maybe give that we are not running on mobile devices, this is maybe over thinking and netstandard2 might make it easier for development by having more apis available.

Reply:
I was thinking that the more we match PowerShell Core the better.

Reply:
Ok, I don't mind too much if you prefer that but changing to `netstandard2.0` should be done as a separate PR.